intro-to-dev-environment/intro-to-computer-memory.md

```diff
@@ -0,0 +1,166 @@
+# Intro to Computer Memory
+
+## Goal
+
+The goal of this lesson is to introduce computer memory. We believe that having an introduction to the stack and heap will allow us to better understand and debug memory allocation, variables, and references vs. values moving forward.
+
+## Introduction
+
+Computers are programmable machines that carry out sequences of instructions and operations. All of the modern computing devices that will run our code will have a memory component. When our code runs, our computers use this memory component to hold onto information like values and variables.
+
+What does it mean when our Activity Monitor says a program takes 100% CPU utilization? What do we mean when we say "memory"? Learning a few things about computer memory can help us be more comfortable debugging.
```

Comment:
I imagine some students may not even know what an Activity Monitor is... a screenshot + short description might be helpful?